coursefinder/templates/coursefinder/course_finder_narrow_search.html

```diff
@@ -0,0 +1,22 @@
+{% extends "coursefinder/course_finder_base.html" %}
+{% load static %}
+{% load wagtailcore_tags %}
+{% block body_class %}template-course-finder-narrow-search{% endblock %}
+
+{% block question %}
+<h3>{{page.question}}</h3>
+<form>
+  <div class="radio">
+    <input id="uni" type="radio" name="radio-group" value=""> <label for="uni">to a specific uni?</label>
+  </div>
+  <div class="radio">
```

Comment:
Can we hide these two labels - town/city and to near your home?

And use an environment variable to unhide them separately?

extra environment variables:
```
FINDBYCITY
FINDBYPOSTCODE
```
these should default to false if not set in environment